### PR TITLE
Rename Test Source to DataGenerator Source

### PR DESCRIPTION
**Pull request by sreevatsanraman**
Unit test: http://builds.cask.co/browse/CDAP-RBT410

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java [hunk 1]
```diff
@@ -133,7 +134,8 @@ public void testEmptyProperties() throws Exception {
   @Category(SlowTests.class)
   public void testStreamSink() throws Exception {
     long startTime = System.currentTimeMillis();
-    ETLStage source = new ETLStage("Test", ImmutableMap.of(TestSource.PROPERTY_TYPE, TestSource.STREAM_TYPE));
+    ETLStage source = new ETLStage("Test", ImmutableMap.of(DataGeneratorSource.PROPERTY_TYPE,
```
Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java]: should be "DataGenerator"

File: cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java [hunk 2]
```diff
@@ -172,7 +174,8 @@ public void testStreamSink() throws Exception {
   @Test
   @SuppressWarnings("ConstantConditions")
   public void testTableSink() throws Exception {
-    ETLStage source = new ETLStage("Test", ImmutableMap.of(TestSource.PROPERTY_TYPE, TestSource.TABLE_TYPE));
+    ETLStage source = new ETLStage("Test", ImmutableMap.of(DataGeneratorSource.PROPERTY_TYPE,
```
Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java]: should be "DataGenerator"

File: cdap-docs/application-templates/source/etl/templates/sources/real-time/datagenerator.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@
     :copyright: Copyright © 2015 Cask Data, Inc.
 
 ===============================
-Sources: Real-time: Test 
+Sources: Real-time: DataGenerator 
 ===============================
```
Comment (albertshau) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/datagenerator.rst]: extend ==== to go up to DataGenerator

Comment (albertshau) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/datagenerator.rst]: you will also need to update the index.rst file

Comment (sreevatsanraman) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/datagenerator.rst]: Fixed

File: cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java (hunk 2 above)
Comment (sreevatsanraman) [inline cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java]: Fixed

File: cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java (hunk 1 above)
Comment (sreevatsanraman) [inline cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java]: Fixed

**Comment by albertshau** [comment]
still need to update index.rst. otherwise lgtm

File: cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst [hunk 4]
```diff
@@ -12,5 +12,5 @@ Sources: Real-time
     AmazonSQS <amazonsqs>
     JMS <jms>
     Kafka <kafka>
-    Test <test>
+    DataGenerator <datagenerator>
```
Comment (albertshau) [inline cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst]: move up (list is alphabetical)

**Comment by albertshau** [comment]
one comment, then lgtm

**Comment by albertshau** [comment]
lgtm

**Pull request merged**